bcda/cclf/cclf.go

```diff
@@ -148,27 +147,12 @@ func importCCLF0(ctx context.Context, fileMetadata *cclfFileMetadata) (map[strin
 }
 
 func importCCLF8(ctx context.Context, fileMetadata *cclfFileMetadata) error {
-	err := importCCLF(ctx, fileMetadata, func(fileID uint, b []byte, db *gorm.DB) error {
-		close := metrics.NewChild(ctx, "importCCLF8-benecreate")
-		defer close()
-		const (
-			mbiStart, mbiEnd   = 0, 11
-			hicnStart, hicnEnd = 11, 22
-		)
-		cclfBeneficiary := &models.CCLFBeneficiary{
-			FileID: fileID,
-			MBI:    string(bytes.TrimSpace(b[mbiStart:mbiEnd])),
-			HICN:   string(bytes.TrimSpace(b[hicnStart:hicnEnd])),
-		}
-		err := db.Create(cclfBeneficiary).Error
-		if err != nil {
-			fmt.Println("Could not create CCLF8 beneficiary record.")
-			err = errors.Wrap(err, "could not create CCLF8 beneficiary record")
-			log.Error(err)
-			return err
-		}
-		return nil
-	})
+	importer := &cclf8Importer{
+		logger:            log.StandardLogger(),
+		maxPendingQueries: utils.GetEnvInt("STATEMENT_EXEC_COUNT", 100000),
```

Comment:
Can we bump this to 200,000?  This will ensure that all current CCLFs get loaded in one shot; and if we run into problems, we can tweak this value.

Reply:
> Does this awesome performance enhancement invalidate this ticket: https://jira.cms.gov/browse/BCDA-3553
> With the performance being <30 seconds (vs ~7 mins), I'd like to see us exercise CCLF Ingestion on each test for awhile.

I think it's a great idea to defer the smoke test changes for a little while at least. I still think we'll get some good benefits from reducing the scope of the ingest work. Definitely agreed that it's valuable since we've made pretty drastic changes to the CCLF ingest process and we should leave it in place for now.